package/src/components/Attachment/GallaryImage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ImageBackground, ImageProps } from 'react-native';
+
+import { makeImageCompatibleUrl } from '../../utils/utils';
+
+export const GalleryImage: React.FC<
+  Omit<ImageProps, 'height' | 'source'> & {
+    uri: string;
+  }
+> = (props) => {
+  const { uri, ...rest } = props;
+
+  return (
+    <ImageBackground
+      {...rest}
+      accessibilityLabel='gallery-image'
+      source={{
+        uri: makeImageCompatibleUrl(uri),
+      }}
+      testID='image-attachment-single'
```

Comment:
Do we still need this `testId` and if so is it the best naming? 
We should always prefer querying components by what is visible to the user, text, labels etc.